### ISSUE TITLE
Perform linebreaks in note title - not in buttons

### ISSUE THREAD

**Issue by budachst** [brantje/nextnote#103]
If a note titel becomes very long, it's not the title that gets wrapped, but instead the buttons get wrapped under  one another. Either fade out the note title or perform a wrapping on that, but keep the buttons on one line.

![grafik](https://user-images.githubusercontent.com/10184884/38720679-78a18860-3ef7-11e8-89d9-ba6fc356af9a.png)

**Comment by budachst** [comment]
I am afraid that PR 108 doesn't resolve this issue. After pulling master, the buttons are still wrapped instead of the note title.

**Comment by trurli** [comment]
Just to make sure, did you clean the browser cache?
If the issue persists, which browser are you using?

**Comment by budachst** [comment]
Yes - I also checked with two different browsers (Safari/Firefox)

**Comment by trurli** [comment]
You're right, it does not work out of the box when checking out a clean copy. 

It is fixed (in theory), but I did not commit the file templates.js which is generated by html2js during the build. If you've got grunt, you can regenerate it yourself to make it work.

@brantje I'll make a new PR for that or is it unnecessary?

**Comment by brantje** [comment]
Did you run grunt?
